.github/workflows/pip-audit.yml

```diff
@@ -26,4 +26,4 @@ jobs:
     - name: Install
       run: python -m pip install .
     - name: Run pip-audit
-      uses: trailofbits/gh-action-pip-audit@v1.0.0
\ No newline at end of file
+      uses: pypa/gh-action-pip-audit@v1.0.2
```

Comment:
Hello @MightyNerdEric, i am curious in this change - how may I verify it's effect in my local copy oh the repo?